qiskit/transpiler/__init__.py

```diff
@@ -40,629 +41,787 @@
+            (2,): InstructionProperties(error=.00002, duration=6e-8)
+        }
+    )
+    target.add_instruction(
+        RYGate(Parameter('theta')),
+        {
+            (1,): InstructionProperties(error=.00001, duration=5e-8),
+            (2,): InstructionProperties(error=.00002, duration=6e-8)
+        }
+    )
+    target.add_instruction(
+        RXGate(Parameter('theta')),
+        {
+            (1,): InstructionProperties(error=.00001, duration=5e-8),
+            (2,): InstructionProperties(error=.00002, duration=6e-8)
+        }
+    )
+    target.add_instruction(
+        CZGate(),
+        {
+            (1, 2): InstructionProperties(error=.0001, duration=5e-7),
+            (2, 0): InstructionProperties(error=.0001, duration=5e-7)
+        }
+    )
+    target.add_instruction(
+        Measure(),
+        {
+            (0,): InstructionProperties(error=.001, duration=5e-5),
+            (1,): InstructionProperties(error=.002, duration=6e-5),
+            (2,): InstructionProperties(error=.2, duration=5e-7)
+        }
+    )
+    print(target)
+
+This :class:`~.Target` represents a 3 qubit QPU that supports :class:`~.CXGate` between qubits
+0 and 1, :class:`~.UGate` on qubits 0 and 1, :class:`~.RZGate`, :class:`~.RXGate`,
+and :class:`~.RYGate` on qubits 1 and 2, :class:`~.CZGate` between qubits 1 and 2, and qubits
+2 and 0, and :class:`~.Measure` on all qubits.
+
+There are also specific data structures to represent a specific subset of information from the
+:class:`~.Target`. For example, the :class:`~.CouplingMap` class is used to solely represent the
+connectivity constraints of a backend as a directed graph. A coupling map can be generated from
+a :class:`~.Target` using the :meth:`.Target.build_coupling_map` method. These data structures
+typically pre-date the :class:`~.Target` class but are often used with transpiler passes that do
+not work natively with a :class:`~.Target` instance or with backends that aren't using the latest
+:class:`~.BackendV2` interface. However, they do also offer more targeted interaction methods
+with the limited view they provide. For example, :class:`~.CouplingMap` provides methods for
+analyzing the connectivity graph, such as :meth:`.CouplingMap.shortest_undirected_path`, which
+can be useful if analysis of the connectivity constraints of a backend is desired.
+
+For example, if we wanted to visualize the :class:`~.CouplingMap` for the
+example 3 qubit :class:`~.Target` above:
+
+.. jupyter-execute::
+
+    target.build_coupling_map().draw()
+
+This shows the global connectivity of the :class:`~.Target` which is the
+combination of the supported qubits for :class:`~.CXGate` and :class:`~.CZGate`. To
+see the individual connectivity uou can pass the operation name to
+:meth:`.CouplingMap.build_coupling_map`:
+
+.. jupyter-execute::
+
+    target.build_coupling_map('cx').draw()
+
+.. jupyter-execute::
+
+    target.build_coupling_map('cz').draw()
 
 .. _transpiler_supplemental:
 
```

Comment:
This link now points to a title "Transpiler Stage Details".  Do we want to update it?  (Changing it should reveal any places in the documentation that would link to incorrect text now as well.)

Reply:
That's a good point, the only thing that references that I know about is the alap and asap scheduling passes. I'll update this ref name and those references (which I'll redirect to the scheduling subsection now instead of the top level section).

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9087/commits/e82fe39745eed35605f36d63a61767202a3ea8d0